### ISSUE TITLE
Filesize: Difference between local phantomjs and your hosted version

### ISSUE THREAD

**Issue by toovy** [ebdrup/html2pdf.it#31]
Hi,
when I convert a page with your site html2pdf.it it is really small as text seems to be rendered as text, I can select it afterwards with Adobe PDF selection tools. When I clone this repo and use it on my mac with phantomjs 2.0 I got a huge PDF file size. When I try to select text in a PDF viewer I can't, I guess the text is rendered as image - that would explain the huge filesize.

So the question is: which phantomjs version is your server using und where do you host? Heroku?

This might be related to https://github.com/ariya/phantomjs/issues/12685.

Thanks, BR, toovy

**Comment by ebdrup** [comment]
Phantom 1.9.x

**Comment by toovy** [comment]
This seems to be a MacOs Phantomjs issue, not related to versions 1.9x or 2.0 . Both versions will produce very large PDFs on my MacOs. Both versions will produce slim PDFs on Heroku. So I consider this as solved.

For those interested in upgrading to a phantomjs 2.0 on heroku: 

I built a 2.0 version for Heroku using this build pack https://github.com/srbartlett/heroku-buildpack-phantomjs-2.0 . Then I deleted /bin/phantompdf and removed following lines in pdf.js:

    if (process.platform !== 'darwin') {
    	pdfExecutable = path.resolve(path.join('bin', pdfExecutable));
    }

From then on phantomjs 2.0 is used to generate the PDFs.

BR, toovy
